src/net/if_.rs

```diff
@@ -21,73 +21,307 @@ pub fn if_nametoindex<P: ?Sized + NixPath>(name: &P) -> Result<c_uint> {
 libc_bitflags!(
     /// Standard interface flags, used by `getifaddrs`
     pub struct InterfaceFlags: libc::c_int {
+        // bitmask: 0x01
         /// Interface is running. (see
         /// [`netdevice(7)`](http://man7.org/linux/man-pages/man7/netdevice.7.html))
         IFF_UP;
+
+        // bitmask: 0x02
         /// Valid broadcast address set. (see
         /// [`netdevice(7)`](http://man7.org/linux/man-pages/man7/netdevice.7.html))
         IFF_BROADCAST;
+
+        // bitmask: 0x04
         /// Internal debugging flag. (see
         /// [`netdevice(7)`](http://man7.org/linux/man-pages/man7/netdevice.7.html))
         IFF_DEBUG;
+
+        // bitmask: 0x08
         /// Interface is a loopback interface. (see
         /// [`netdevice(7)`](http://man7.org/linux/man-pages/man7/netdevice.7.html))
         IFF_LOOPBACK;
+
+        // bitmask: 0x10
         /// Interface is a point-to-point link. (see
         /// [`netdevice(7)`](http://man7.org/linux/man-pages/man7/netdevice.7.html))
         IFF_POINTOPOINT;
+
+        // bitmask: 0x20
         /// Avoid use of trailers. (see
         /// [`netdevice(7)`](http://man7.org/linux/man-pages/man7/netdevice.7.html))
         #[cfg(any(target_os = "android",
+                  target_os = "bitrig",
```

Comment:
There's no need to worry about bitrig anymore.  A future nix PR will probably remove all references.  For now, it doesn't really matter whether or not you leave it in.
https://www.phoronix.com/scan.php?page=news_item&px=Bitrig-Fork-Dead

Reply:
Okay, thanks for the hint.